button.html

```diff
@@ -14,12 +14,12 @@ <h3> Why I'm Creating a website </h3>
 	<h4> You can listen to deadmouse music all day :p</h4>
 
 
-	<button type="button"
-onclick="document.getElementById('demo').innerHTML = Date()">
-Click me to be epic
-<p id= "demo"></p>
+	<button type="button" onclick="document.getElementById('demo').innerHTML = Date()"> <p Click me to be epic id= "demo">></p
+
+
+
 <script>  getElementById("demo").innerHTML="Hi dad!";
```

Comment:
Two things here: 1. Move the `script` block to the header of the file and 2. Create a function called `myFunction` that contains the `innerHTML` code you have inside it.  Then in your button you just call the `myFunction` to do everything.